src/app/globals.css

```diff
@@ -52,6 +56,33 @@ button {
   font-size: 28px;
   font-family: var(--font-mono);
 }
+
+.toggle-switch {
+  position: relative;
+  width: 50px;
+  height: 25px;
+  background-color: #ccc;
+  border-radius: 25px;
+  cursor: pointer;
+}
+
+.switch {
+  position: absolute;
+  width: 23px;
+  height: 23px;
+  border-radius: 50%;
+  background-color: #fff;
+  transition: 0.3s;
+}
+
+.light {
+  left: 2px;
+}
+
+.dark {
+  left: 25px;
+}
+
```

Comment:
for classes tied to a component, do not put them in a global file. they must be only tied to the component otherwise you will not only clutter the `globals.css` but also introduce tightly coupled code. `globals.css` is only meant for styling to be applied across the page like font sizes, colors, etc.